package.json

```diff
@@ -8,7 +8,7 @@
     "test-integration": "casperjs test ./test/integration/test.*.js",
     "test-integration-runner": "NODE_ENV=test node casper-runner.js",
     "test": "npm run lint && npm run test-unit && npm run test-integration-runner",
-    "test-unit": "istanbul cover ./node_modules/mocha/bin/_mocha test/unit",
+    "test-unit": "./node_modules/mocha/bin/_mocha --compilers js:babel-core/register test/views/MentionsAndSentiments test/unit",
```

Comment:
does this work if we just do `./node_modules/mocha/bin/_mocha test/views test/unit`? then create a file `./test/mocha.opts` with the `--compilers js:babel-core/register` option? see http://mochajs.org/#mochaopts